docs/UserGuide.md

```diff
@@ -224,6 +224,13 @@ Format: `exit`
 
 Exports flashcards from LingoGO! to a CSV file.
 
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes about opening CSV files with Excel:**<br>
+* We advise changing the default settings so that foreign language is properly displayed with Excel.
+[Please refer here for detailed instructions.](https://www.itg.ias.edu/content/how-import-csv-file-uses-utf-8-character-encoding-0)
+
+</div>
+
 Format: `export FILE_NAME`
```

Comment:
For import, the parameter is `CSV_FILE_NAME`, so maybe we should standardize to use that

Reply:
#157 arose most likely because the reviewer did not provide the correct file path. The issue is addressed in this PR because there is now no need for users to specify the file path with the changed UG.